runtime/runtime.m

```diff
@@ -623,21 +632,18 @@ -(void) xamarinSetFlags: (enum XamarinGCHandleFlags) flags;
 	return (MonoException *) mono_exception_from_name_msg (mono_get_corlib (), "System", "Exception", msg);
 }
 
-typedef struct {
-	MonoObject object;
-	MonoMethod *method;
-	MonoString *name;
-	MonoReflectionType *reftype;
-} PublicMonoReflectionMethod;
-
 MonoMethod *
 xamarin_get_reflection_method_method (MonoReflectionMethod *method)
 {
 	// COOP: Reads managed memory, needs to be in UNSAFE mode
 	MONO_ASSERT_GC_UNSAFE;
 	
+#if defined (CORECLR_RUNTIME)
+	xamarin_assertion_message ("The method %s it not implemented yet for CoreCLR", __func__);
```

Comment:
```suggestion
	xamarin_assertion_message ("The method %s is not implemented yet for CoreCLR", __func__);
```
Maybe typo?

Reply:
ah nice catch! I had to blink several times (on email) before I could spot the typo haha